media/css/firefox/whatsnew/whatsnew-93.scss

```diff
@@ -0,0 +1,182 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/notification-bar';
+@import 'includes/header';
+
+
+//* -------------------------------------------------------------------------- */
+// Main content
+.wnp-content-main {
+    padding: 0;
+    text-align: center;
+}
+
+.wnp-main-image {
+    display: block;
+    max-width: 800px;
```

Comment:
Looks like this is preventing the main image in http://localhost:8000/en-US/firefox/93.0/whatsnew/en/?v=1 from resizing with the browser. Might need `width: 100%;` as well?